### PR TITLE
docs: updating eks section for date with dkp 2.2

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

n/a

## Description of changes being made

Updating to keep pace with eks and dkp 2.2

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4249.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by msdolbey** [comment]
Adding DO NOT MERGE until @dkoshkin has a chance to add some comments

**Review by mwrightd2 — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/delete/index.md [hunk 1]
```diff
@@ -7,91 +7,7 @@ excerpt: Delete the Kubernetes cluster and clean up your environment
...
-    ```
-
-    ```sh
-    cluster.cluster.x-k8s.io/eks-example condition met
-    ```
-
-## Delete the workload cluster
+## Delete the EKS cluster
 
 1.  Make sure your AWS credentials are up to date. Refresh the credentials using this command:
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/delete/index.md:12]:
```suggestion
1.  Ensure your AWS credentials are up to date. Refresh the credentials using this command:
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/delete/index.md [hunk 2]
```diff
@@ -104,7 +20,7 @@ If you did not make your workload cluster self-managed, as described in [Make Ne
     Before deleting the cluster, dkp deletes all Services of type LoadBalancer on the cluster. Each Service is backed by an AWS Classic ELB. Deleting the Service deletes the ELB that backs it.
     To skip this step, use the flag `--delete-kubernetes-resources=false`.
 
-    <p class="message--note"><strong>NOTE: </strong>Do not skip this step if the VPC is managed by DKP. When dkp deletes cluster, it deletes the VPC. If the VPC has any EKS Classic ELBs, EKS does not allow the VPC to be deleted, and dkp cannot delete the cluster.</p>
+    <p class="message--note"><strong>NOTE: </strong>Do not skip this step if the VPC is managed by DKP. When DKP deletes cluster, it deletes the VPC. If the VPC has any EKS Classic ELBs, EKS does not allow the VPC to be deleted, and dkp cannot delete the cluster.</p>
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/delete/index.md:23]:
```suggestion
    <p class="message--note"><strong>NOTE: </strong>Do not skip this step if the VPC is managed by DKP. When DKP deletes the cluster, it deletes the VPC. If the VPC has any EKS Classic ELBs, EKS does not allow the VPC to be deleted, and DKP cannot delete the cluster.</p>
```

**Review by gvasquezvargas — COMMENTED** [review]
Nothing big, just two minor suggestions.

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md [hunk 3]
```diff
@@ -77,12 +77,12 @@ Tips:
     This creates a unique name every time you run it, so use the command with forethought.
 
     ```bash
-    CLUSTER_NAME=$(whoami)-aws-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
+    CLUSTER_NAME=$(whoami)-eks-cluster-$(LC_CTYPE=C LC_COLLATE=C tr -dc 'a-z0-9' < /dev/urandom | fold -w 5 | head -c 5)
```
Comment (dkoshkin) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md:80]: Lets keep this what it was before to be consistent with the other pages
```suggestion
    CLUSTER_NAME=$(whoami)-eks-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md [hunk 4]
```diff
@@ -131,7 +131,7 @@ Tips:
 
     <p class="message--note"><strong>NOTE: </strong>This kubeconfig is temporary and needs to be renewed either by running the above command with updated credentials and rerunning, or by running the AWS CLI command:</p>
 
-    ```yaml
+    ```bash
     aws eks --region us-west-2 update-kubeconfig --name default_<name-of-cluster>-control-plane
     ```
```
Comment (dkoshkin) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md:136]: Please remove this the note and the `aws eks` command here, it is no longer temporary.

Please also add a prereq to install https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html

**Review by Colin1841 — APPROVED** [review]

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md (hunk 3 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md]: boo lol

**Pull request merged**